sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/MathFunctionsSuite.scala

```diff
@@ -252,4 +254,33 @@ class MathFunctionsSuite extends SparkFunSuite with ExpressionEvalHelper {
       null,
       create_row(null))
   }
+
+  test("round test") {
+    val domain = -16 to 16
+    val doublePi = math.Pi
+    val stringPi = "3.141592653589793"
+    val intPi = 314159265
+    val bdPi = BigDecimal(31415926535897932L, 10)
+
+    domain.foreach { scale =>
+      checkEvaluation(Round(doublePi, scale),
+        BigDecimal.valueOf(doublePi).setScale(scale, RoundingMode.HALF_UP).toDouble, EmptyRow)
```

Comment:
yes, this can be done, but I prefer to remain these to just follow other maths test case. btw,I guess it would be obscure  to read.